toolchains/cc/cc.bzl

```
@@ -221,7 +221,7 @@ def _nixpkgs_cc_toolchain_config_impl(repository_ctx):
             "%{coverage_compile_flags}": get_starlark_list(info.coverage_compile_flags),
             "%{coverage_link_flags}": get_starlark_list(info.coverage_link_flags),
             "%{supports_start_end_lib}": repr(info.supports_start_end_lib),
-            "%{extra_flags_per_feature}": repr(info.extra_flags_per_feature),
+            "%{extra_flags_per_feature}": repr(dict(info.extra_flags_per_feature)),
```

`dict()` expects an iterable of pairs as its first argument. `info.extra_flags_per_feature` is a list of strings. This will only work out if this list is empty. So, this is a bit complicated way of always writing out `{}`.

Is there a way forward for the `EXTRA_FLAGS_PER_FEATURE` in `cc.nix` to be useful? Otherwise I'd rather remove it until there is a use case for it.

---

Would you suggest we simply do

```suggestion
            "%{extra_flags_per_feature}": repr({}),
```

and remove the rest until there is a concrete usecase?

---

Yes, even just using `"{}"` would do.

---

Done